
From: <janitor@sternwelten.at>

 Patches to remove some old ifdefs.
 remove most of the #include <linux/version.h>
 kill compat cruft like #define ahd_pci_set_dma_mask pci_set_dma_mask

Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/hamradio/dmascc.c |    1 -
 1 files changed, 1 deletion(-)

diff -puN drivers/net/hamradio/dmascc.c~janitor-remove-old-ifdefs-dmascc drivers/net/hamradio/dmascc.c
--- 25/drivers/net/hamradio/dmascc.c~janitor-remove-old-ifdefs-dmascc	Thu Sep 23 15:38:52 2004
+++ 25-akpm/drivers/net/hamradio/dmascc.c	Thu Sep 23 15:38:52 2004
@@ -37,7 +37,6 @@
 #include <linux/rtnetlink.h>
 #include <linux/sockios.h>
 #include <linux/workqueue.h>
-#include <linux/version.h>
 #include <asm/atomic.h>
 #include <asm/bitops.h>
 #include <asm/dma.h>
_
